tutorials/cp-starter-digitalexp-mobilecards-1-setup/cp-starter-digitalexp-mobilecards-1-setup.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Configure Initial Setup for SAP Mobile Cards
+description: Define a destination in SAP Cloud Platform Mobile Services cockpit to establish connectivity to a SAP SuccessFactors sandbox API.
+auto_validation: true
+primary_tag: products>sap-mobile-cards
+tags: [ tutorial>beginner, operating-system>ios, operating-system>android, topic>mobile, products>sap-cloud-platform, products>sap-mobile-cards, software-product-function>sap-cloud-platform-mobile-services ]
+time: 10
+author_name: Jitendra Kansal
+author_profile: https://github.com/jitendrakansal
+---
+
+## Details
+We will create a destination to connect to the SAP SuccessFactors API data source from SAP Cloud Platform Mobile Services.
+
+### You will learn
+  - How to access the SAP Cloud Platform Mobile Services cockpit
+  - How to establish connectivity to a SAP SuccessFactors API
+
+---
+
+[ACCORDION-BEGIN [Step 1: ](Understand the feature - SAP Mobile Cards)]
+
+SAP Mobile Cards is a feature within SAP Cloud Platform Mobile Services which provides our customers access to a micro application platform to publish data into a consumer grade wallet or passbook-style app. It allows companies to quickly create simple, yet highly valuable quick-win apps.
+
+You can find more details on the [SAP Mobile Cards developer page](https://developers.sap.com/topics/mobile-cards.html).
+
+![MobileCards](MobileCards.png)
+
+[DONE]
+[ACCORDION-END]
+[ACCORDION-BEGIN [Step 2: ](Navigate to the SAP Mobile Services cockpit)]
+
+On the SAP Cloud Platform Trial landing page, choose **Enter your Trial Account**.
+
+![MobileCards](img_0.png)
+
+>The trial account should contain one `subaccount` and `space`.
+
+Choose **trial**.
+
+![cf Trial](img_2.png)
```

Comment:
this photo in test description is not confidential enought. space name and region should be blurred.

Reply:
Done